### PR TITLE
add dummy property to test flux config

### PR DESCRIPTION
**Pull request by taomoh**
### JIRA link ###

https://tools.hmcts.net/jira/browse/ROC-9708

### Change description ###

Add dummy property to test flux config

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: charts/civil-citizen-ui/values.preview.template.yaml [hunk 1]
```diff
@@ -1,4 +1,6 @@
-nodejs:
-  # Don't modify below here
-  image: ${IMAGE_NAME}
-  ingressHost: ${SERVICE_FQDN}
+    nodejs:
```
Comment (timja) [inline charts/civil-citizen-ui/values.preview.template.yaml:1]: indenting is broken in this file, remove the initial indent

File: config/custom-environment-variables.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (timja) [inline config/custom-environment-variables.json:1]: most other apps use yaml as its easier to read especially for config files.

Comment (taomoh) [reply config/custom-environment-variables.json:1]: Can you provide an example of an app which uses yaml please? When I switched the format, the application broke locally. My understanding is that node.js expects environment variables to be defined in json files under config/ folder

**Review by vinothon84 — APPROVED** [review]
PR looks good and apprved now.

**Pull request merged**